EIPS/eip-4675.md

```diff
@@ -0,0 +1,218 @@
+```solidity
+/**
+    @notice Handle the receipt of an NFT
+    @param _operator The address which called `safeTransferFrom` function
+    @param _from The address which previously owned the token
+    @param _tokenId The NFT identifier which is being transferred
+    @param _data Additional data with no specified format
+    @return `bytes4(keccak256("onERC721Received(address,address,uint256,bytes)"))`
+*/
+function onERC721Received(address _operator, address _from, uint256 _tokenId, bytes calldata _data) external pure returns (bytes4);
+```
+
+## Rationale
+
+**Metadata**
+
+The `symbol()` & `name()` function was not included since the majority of users can just fetch it from the originating NFT contract. Also, copying the name & symbol every time when token gets added might place a lot of redundant bytecode on the Ethereum blockchain. 
+However, according to the need and design of the project it could also be added to each token type by fetching the metadata from the NFT contract.
+
+**Design**
+
+Most of the decisions made around the design of this ERC were done to keep it as flexible for diverse token design & architecture.
+These minimum requirement for this standard allows for each project to determine their own system for minting, governing, burning their MFNFT tokens depending on their programmable architecture.
+
+## Backwards Compatibility
+
+To make this standard compatible with existing standards, this standard `event` & `function` names are identical with ERC-20 token standard with some more `events` & `functions` to add token type dynamically.
+
+Also, the sequence of parameter in use of `_id` for distinguishing token types in `functions` and `events` are very much similar to ERC-1155 Multi-Token Standard.
+
+Since this standard is intended to interact with the EIP-721 Non-Fungible Token Standard, it is kept purposefully agnostic to extensions beyond the standard in order to allow specific projects to design their own token usage and scenario.
+
+## Test Cases
+
+Reference Implementation of MFNFT Token includes test cases written using hardhat. (Test coverage : 100%)
+
+## Reference Implementation
+[MFNFT - Implementation](../assets/eip-4675/)
+
+## Security Considerations
+
+To fractionalize an already minted NFT, it is evident that ownership of NFT should be given to token contracts before fractionalization.
+In the case of fractionalizing NFT, the token contract should thoroughly verify the ownership of NFT before fractionalizing it to prevent tokens from being a separate tokens with the NFT.
+
+If an arbitrary account has the right to call `setParentNFT()` there might be a front-running issue. The caller of `setParentNFT()` might be different from the real NFT sender. 
+To prevent this issue, implementors should just allow **admin** to call, or fractionalize and receive NFT in an atomic transaction similar to flash loan(swap).
+
+## Copyright
+Copyright and related rights waived via [CC0](../LICENSE.md).
+
+## Citation
```

Comment:
Please remove this section, it is added automatically on the website `eips.ethereum.org`.

Reply:
@PowerStream3604, this comment is on the "Citation" section. The "Copyright" section is required.

Reply:
@lightclient , I updated the file as requested.
Sorry, I misunderstood it.